### PR TITLE
Add new career path

### PR DESCRIPTION
**Pull request by Taimoorkhan1122**
Hi, this is Taimoor, This is my first pull request really excited about this. I have added a new career i.e IoT will be updating it as I found more related info. Kindly review my pull request and if I have done anything wrong please let me know...

### CODE REVIEW

**Comment by Rubix982** [comment]
Hi, @Taimoorkhan1122 ! Sure, I'll look into it.

**Review by Rubix982 — CHANGES_REQUESTED** [review]
Really appreciate the work you've done. There do need to be some changes that need to be done,

1. Is it better if we name the topic as simply `IoT`, rather than `IoT Development`? IoT captures a lot more information, not simply just the development of it. It can, for example, be easier to talk about future advancements, researches, conferences, and academic advances in this field.
2. The template you've used is for the `learning-resources`, not for `career-paths`. Please check out [this](https://softdevpk.com/career-paths/#please-read-first) for the contribution template for `career-paths`. Also, is it easy for your to understand? Or do you think that there should be more brevity to the the page I linked above.
3. Please check out the `edit-template-career-paths` branch. I made an edit to the template for `disqus` relevant format matter.

Thanks!

**Review by Rubix982 — CHANGES_REQUESTED** [review]
Some descriptions are sort of vague,

1. `Applications Of Iot` should explain, i) Why it matters, ii) What fields exactly use it right now. The numbers are really good editions, but it doesn't explain why IoT is growing, just that IoT is growing for some reason.
2. What are `Iot Verticals`? What exactly do they mean? Why does it matter? More explanation seems to be desired here.

**Review by Rubix982 — CHANGES_REQUESTED** [review]
Place and link to the image in the `assets/img/` folder, not the `images/` folder. The `images/` folder is for icons used through out the website, like the logo. The `assets/img/` folder is for images used in pages and posts.

**Review by Rubix982 — CHANGES_REQUESTED** [review]
There hasn't been a discussion of making a `contributors.md` in each of the career paths. 

Please take a look, @azmaktr

**Comment by Rubix982** [comment]
All in all, good efforts! Love it. IoT is a pretty good and growing field as well - hopefully we can make this as a career path reality as well. 
Thanks! :D

File: career-paths/IoT-dev/index.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+- [Sources](#sources)
+  - [Books / Documentation](#books--documentation)
+  - [Videos](#videos)
+  - [MISC](#misc)
+
+## What is IoT 
+![IoT image](../../images/The-Internet-of-Things--IoT--Tech-Stack-Explained.jpg)
+IoT stands for **I**nternet **O**f **T**hings. IoT refers to billions of physical devices around the world that are connected to the internet collecting, sharing and analyzing data. Thanks to cheap processors and wireless networks, it's possible to turn anything, from a pill to an aeroplane to a self-driving car into part of the IoT
+<br>
+The Internet Of Things is network of physical devices (Things) that are embedded with Sensors, Actuators , Software and other electronics over the internet through IP connectivity to integerate the physical world into our digital systems, Providing efficiency improvements and economical benifits.
```
Comment (dani2819) [inline career-paths/IoT-dev/index.md:22]:
```suggestion
The Internet Of Things is a network of physical devices (Things) that are embedded with sensors, actuators, software and other electronics over the internet through IP connectivity to integrate the physical world into our digital systems, Providing efficiency improvements and economical benefits.
```

File: career-paths/IoT-dev/index.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## Application of IoT
+IoT is one the key factors of Fourth Industrial Revolution.
```
Comment (dani2819) [inline career-paths/IoT-dev/index.md:25]:
```suggestion
IoT is one of the key factors of the fourth Industrial Revolution. 
```

File: career-paths/IoT-dev/index.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+- Smart Building
+- Healthcare
+- Energy
+- Transportation and Smart Vehicles
+- Logistics and Inventory Management
+- Retail
+- Agriculture
+- Military
+
+## Technology Stack for Iot
```
Comment (dani2819) [inline career-paths/IoT-dev/index.md:43]:
```suggestion
## Technology Stack for IoT
```

File: career-paths/IoT-dev/index.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+- Logistics and Inventory Management
+- Retail
+- Agriculture
+- Military
+
+## Technology Stack for Iot 
+The IoT stack is rapidly developing and maturing into the Thing Stack. This Thing Stack consists of three technology layers: sensors, microcontrollers and internet connectivity, and service platforms.
+- **Layer One** - Sensors are embedded in objects (Things) or the physical environment to capture information and events from our environment, like a temperature sensor.
+
+- **Layer Two** - Microcontrollers and internet connectivity share information captured by sensors within your IoT objects and act based on this information to change the environment, like smart AC maintaining temperature of room.
```
Comment (dani2819) [inline career-paths/IoT-dev/index.md:47]:
```suggestion
- **Layer Two** - Microcontrollers and internet connectivity share information captured by sensors within your IoT objects and act based on this information to change the environment, like smart AC maintaining the temperature of the room.
```

File: career-paths/IoT-dev/index.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+- **Layer Three** - Through the aggregation and analysis of data, service platforms cater to your customers. Service platforms also control your IoT product's end-to-end experience and enable your customers to define system rules and update firmware.
```
Comment (dani2819) [inline career-paths/IoT-dev/index.md:49]:
```suggestion
- **Layer Three** - Through the aggregation and analysis of data, service platforms cater to your customers. Service platforms also control your IoT product's end-to-end experience and enable your customers to define system rules and update the firmware.
```

**Review by dani2819 — CHANGES_REQUESTED** [review]
Spell/Grammatical mistakes need to be fixed. Otherwise, LGTM.

**Comment by azmaktr** [comment]
@Taimoorkhan1122  can you please commit the suggestion so we approve and merge? :)

File: career-paths/IoT-dev/contributors.md [hunk 6]
```diff
@@ -0,0 +1 @@
+Taimoor khan
```
Comment (husyn) [inline career-paths/IoT-dev/contributors.md:1]: Do we need this file? @Rubix982 @azmaktr

File: career-paths/IoT-dev/index.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
+---
+title: IoT Development
+layout: page
+---
+
+# IoT Development
+
+# Table Of Contents
+- [What IoT](#what-is-Iot)
```
Comment (husyn) [inline career-paths/IoT-dev/index.md:9]: Heading should be "What is IoT". Word "is" is missing

File: career-paths/IoT-dev/contributors.md (hunk 6 above)
Comment (Rubix982) [reply career-paths/IoT-dev/contributors.md:1]: Currently we haven't had a discussion if we wanted to make a contributors file, so I'm unsure.
We can put this idea in the backlog - however, if we want to pursue with this idea. But for now, there has been no talk about this.

**Comment by Rubix982** [comment]
@Taimoorkhan1122 can you take a look at the changes requested? Thank you!

**Comment by husyn** [comment]
Closing this PR without merging as required changes are pending for more than a month. If you wish to complete this PR please re-open it.

**Pull request closed without merging**